xml/snmp_sysdescr.xml

```diff
@@ -5158,6 +5158,121 @@ Copyright (c) 1995-2005 by Cisco Systems
    </fingerprint>
 
    <!--======================================================================
+                              Mac OS X
+   =======================================================================-->
+   <fingerprint pattern="^Darwin (\S*) \S* Darwin Kernel Version 7.(\d+).\d+: .*RELEASE_\S* ?\s* (\S* ?\S*)$">
+      <description>Mac OS X 10.3 (Panther)</description>
+      <example host.name="host.example.com" os.arch="Power Macintosh" os.version.version="0">Darwin host.example.com 7.0.0 Darwin Kernel Version 7.0.0: Wed Sep 24 15:48:39 PDT 2003; root:xnu/xnu-517.obj~1/RELEASE_PPC Power Macintosh</example>
+      <param pos="1" name="host.name"/>
+      <param pos="0" name="os.vendor" value="Apple"/>
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.device" value="General"/>
+      <param pos="3" name="os.arch"/>
+      <param pos="0" name="os.version" value="10.3"/>
+      <param pos="2" name="os.version.version"/>
+      <param pos="0" name="os.certainty" value="0.9"/>
+   </fingerprint>
+
+   <fingerprint pattern="^Darwin (\S*) \S* Darwin Kernel Version 8.(\d+).\d+: .*RELEASE_\w* (\S* ?\S*)$">
```

Comment:
All the `pattern`s from here down are the same (minus the major version), and the one above differs slightly.  I don't see why.  Can you elaborate?  Also, is `\w` the right choice here (that'll only match a-z, A-Z, 0-9 and _), especially given that you use `\S` above as part of the matching of `RELEASE_`?